reference/providers/kubernetes-v2/index.md

```diff
@@ -128,6 +128,14 @@ command](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands
   ConfigMap or Secret to be deployed without appending a new version when the
   contents change, for example.
 
+* `strategy.spinnaker.io/use-source-capacity`
+
+  When set to `'true'` or `'false'`, this overrides the resources replicas size
+  with the currently deployed resource size. This is supported for 
+  deployment, replicaset or stateful set. 
+  This can be used to allow resizing a resource and maintaining the same 
```

Comment:
I would reword to ".. to allow resizing a resource in the Spinnaker UI or with `kubectl` without overriding the new size during subsequent manifest deployments" to make it clear where the resizing can happen.